styles/partials/base-partials/_utils.scss

```diff
@@ -0,0 +1,98 @@
+//not sure this is needed
+// @import "../mixins";
+
+/* alignment
+......................................*/
+.text-top {
+  vertical-align: top !important;
+}
+.text-middle {
+  vertical-align: middle !important;;
+}
+.text-bottom {
+  vertical-align: bottom !important;;
+}
+
+/* display
+......................................*/
+.block {
+  display: block !important;
+}
+.inline {
+  display: inline-block !important;
```

Comment:
Since this `.inline` exists, that entire code block I commented on above is actually unnecessary. Instead of 
```
+.form {
+  fieldset {
+    &.inline {
+      label, .inner-button, input {
+        display: inline-block;
+      }
+    }
+  }
```

which only adds `inline-block`, this class is sufficient. By adding the css piece-by-piece we can get rid of a whole lot of unnecessary rules.